doc/developers/advanced_installation.rst

```diff
@@ -0,0 +1,427 @@
+
+.. _advanced-installation:
+
+===================================
+Advanced installation instructions
+===================================
+
+There are different ways to get scikit-learn installed:
+
+  * Install the version of scikit-learn provided by your
+    :ref:`operating system or Python distribution <install_by_distribution>`.
+    This is the quickest option for those who have operating systems that
+    distribute scikit-learn.
+
+  * :ref:`Install an official release <install_official_release>`. This
+    is the best approach for users who want a stable version number
+    and aren't concerned about running a slightly older version of
+    scikit-learn.
+
+  * :ref:`Install the latest development version
+    <install_bleeding_edge>`. This is best for users who want the
+    latest-and-greatest features and aren't afraid of running
+    brand-new code.
+
+.. note::
+
+    If you wish to contribute to the project, it's recommended you
+    :ref:`install the latest development version<install_bleeding_edge>`.
+
+
+.. _install_official_release:
+
+Installing an official release
+==============================
+
+Scikit-learn requires:
+
+- Python (>= 2.6 or >= 3.3),
+- NumPy (>= 1.6.1),
+- SciPy (>= 0.9).
+
+
+Windows
+-------
+
+First you need to install `numpy <http://numpy.scipy.org/>`_ and `scipy
+<http://www.scipy.org/>`_ from their own official installers.
+
+Wheel packages (.whl files) for scikit-learn from `PyPI
+<https://pypi.python.org/pypi/scikit-learn/>`_ can be installed with the `pip
+<http://pip.readthedocs.org/en/latest/installing.html>`_ utility.
+Open a console and type the following to install or upgrade scikit-learn to the
+latest stable release::
+
+    pip install -U scikit-learn
+
+If there are no binary packages matching your Python version you might
```

Comment:
Fixed (and the other ones). This is just copy and paste of what was under "installation" btw, so these are not really related to the move of the docs.